File: Javascript/Algorithms/1486.js
/**
 * @param {number} n
 * @param {number} start
 * @return {number}
 */
var xorOperation = function(n, start) {
    let res = start;
    for(let i = 1; i < n; i++){
        res ^= start + 2 * i;
    }
    return res;
};

File: Rust/Algorithms/1486.rs
impl Solution {
    pub fn xor_operation(n: i32, start: i32) -> i32 {
        let mut res = start;
        let mut i = 0;
        for i in 1..n {
            res ^= start + 2 * i;
        }
        res
    }
}
